tests/integration/mutation/update/crdt/pncounter_test.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package update
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestMutationUpdate_PNCounterInt_NoError(t *testing.T) {
```

Comment:
todo: similar to the previous comment: "Mutation" is not needed.

Also I've notices that many tests below lack descriptiveness. A test's name should say ideally what it tests (precondition) and the it expects as a result. For most of these tests the result "NoError", which is not quite correct, because we expect specific results.

For this one I suggest something like `TestPNCounterIntUpdate_WithPositiveValue_ShouldIncrement` (btw, not sure I saw tests with negative values)

For `TestMutationUpdate_PNCounterIntWithOverflow_NoError` test I would suggest `TestPNCounterIntUpdate_IfOverflows_StoresMaxInt`

Reply:
> (btw, not sure I saw tests with negative values)

There is a test with negative value.

Reply:
✅